featuremanagement/_defaultfilters.py

```diff
@@ -92,9 +92,7 @@ def _target_group(self, target_user, target_group, group, feature_flag_name):
         group_rollout_percentage = group.get(ROLLOUT_PERCENTAGE_KEY, 0)
         if not target_user:
             target_user = ""
-        audience_context_id = (
-            target_user + "\n" + target_group + "\n" + feature_flag_name + "\n" + group.get(FEATURE_FILTER_NAME_KEY, "")
-        )
+        audience_context_id = target_user + "\n" + feature_flag_name + "\n" + target_group
```

Comment:
Just curious, is this to be consistent with our other providers?

Reply:
Yeah, this is a bug found by the linked tests. https://github.com/microsoft/FeatureManagement-Dotnet/blob/709bbebd89e2923f820f75e651fd39778df26cbe/src/Microsoft.FeatureManagement/Targeting/ContextualTargetingFilter.cs#L119

Reply:
Got it, thanks!